content/TrueCommand/TCGettingStarted/InitialConfig.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Creating the Initial Configuration"
+description: "This article describes how to initially configure TrueCommand."
+aliases:
+weight: 20
+tags:
+- tcinstall
+- tcadmin
+- tcconfig
+- tcresetpwd
+- scaletoptoolbar
+- coretoptoolbar
+---
+{{< toc >}}
+TrueCommand allows you to manage all your TrueNAS systems from one location. This multi-system management increases efficiency and simplifes operations. To get started with the initial configuration of TrueCommand, ensure that the following steps have been completed:
+
+* You have created the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
```

Comment:
convert these to summary steps in an ordered list might be a better way to guide them through the objective, which is to configure using the Administration page:
```suggestion
1. Create the administrator account. See [Creating the Admin Account]({{< relref "/TrueCommand/TCGettingStarted/AdminAccounts.md" >}}) for more information.
```

Reply:
Change made.